packages/web-react/src/components/Header/useHeaderStyleProps.ts

```diff
@@ -5,6 +5,7 @@ import { HEADER_COLOR_DEFAULT, HEADER_ACTIONS_COLOR_DEFAULT } from './constants'
 
 export interface UseHeaderStyleProps {
   color?: HeaderColorType;
+  isActionAtEnd?: boolean;
```

Comment:
I'm not sure if singular is better than (the incorrect) plural `Actions` mismatching the `is` verb…

What do you think?

```suggestion
  isActionsAtEnd?: boolean;
```

or

```suggestion
  hasActionsAtEnd?: boolean;
```

or

```suggestion
  isActionsSlotAtEnd?: boolean;
```